Dockerfile

```diff
@@ -1,16 +1,23 @@
 # Taichi Dockerfile for development
-FROM nvidia/cuda:10.0-devel-ubuntu18.04
-
+ARG UBUNTU
```

Comment:
Let's create a separate Dockerfile for CI purpose. This one is used for building an image with taichi, IIUC. @rexwangcc 

That said, it seems we can create a script so that the two Dockerfiles can share lots of stuff